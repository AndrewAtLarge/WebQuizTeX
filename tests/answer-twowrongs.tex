% should give error as has two \whenWrong's
\documentclass[hidesidemenu]{webquiz}
\title{String comparison for the answer}
\begin{document}
  \begin{question}     % a quiz question
     What is the capital of Australia?
     \answer[string]{Canberra}
     \whenRight Yes, Sydney is the capital of NSW
     \whenWrong No, Melbourne is the capital of Victoria
     \whenWrong Remember to correctly capitalise your answer!
  \end{question}
\end{document}
