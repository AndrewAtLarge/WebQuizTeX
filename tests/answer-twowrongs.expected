! Package webquiz Error: Only one \when wrong per question is allowed!.
